server/command.go

```diff
@@ -31,9 +31,41 @@ const COMMAND_HELP = `* |/gitlab connect| - Connect your Mattermost account to y
 * |/gitlab me| - Display the connected GitLab account
 * |/gitlab settings [setting] [value]| - Update your user settings
   * |setting| can be "notifications" or "reminders"
-  * |value| can be "on" or "off"`
-
+  * |value| can be "on" or "off"
+* |/gitlab webhook list [owner]/repo| - Will list associated group or project hooks.
+* |/gitlab webhook add owner[/repo] [options] [url] [token]|
+  * |options| is a comma-delimited list of one or more the following:
+	 * |*| - or missing defaults to all with SSL verification enabled
+	 * *noSSL - all triggers with SSL verification not enabled.
+	 * PushEvents
+	 * TagPushEvents 
+	 * Comments 
+	 * ConfidentialComments 
+	 * IssuesEvents
+	 * ConfidentialIssuesEvents 
+	 * MergeRequestsEvents 
+	 * JobEvents 
+	 * PipelineEvents 
+	 * WikiPageEvents
+	 * SSLverification
+  * |url| is the URL that will be called when triggered. Defaults to this plugins URL
```

Comment:
What is a use case for setting this not to the plugin URL?

Reply:
creating a webhook  for your CI/CD tooling or another chat or notification service

Reply:
Can you elaborate a bit? Is the exact use case that a user wan't to create a webhook to an different service inside of Mattermost?

Reply:
Yes, An example user story would be a developer creates a new GitLab repo, and wants to get it added to the team's Jenkins server. So the developer messages the Jenkins administrator to get the token and Jenkins URL for the webhook. Then the developer runs the Mattermost Gitlab Plugin command and creates the webhook to Jenkins.